CHANGELOG.md

```diff
@@ -11,19 +11,48 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 <sup>Announced June 2025</sup>
 
-**Added:**
+### Added
 
-- `BillingAccountType` column
-- `SubAccountType` column
+#### New columns
 
-**Changed:**
+- `BillingAccountType`
+- `InvoiceID`
+- `SubAccountType`
+- `PricingCurrency`
+- `PricingCurrencyContractedUnitPrice`
+- `PricingCurrencyEffectiveCost`
+- `PricingCurrencyListUnitPrice`
 
-- `Column Naming and Ordering` attribute includes the following new provisions:
-  - Attribute renamed to `Column Handling`.
-- `CommitmentDiscountType` column updates:
+### Changed
+
+#### Changed columns
+
+- Normative requirements guidelines have been applied to all columns.
+- `CommitmentDiscountType`
   - Must be a consistent, readable display value.
-- `SkuPriceDetails` column updates:
+- `SkuPriceDetails`
   - Created FOCUS-defined properties.
+- `Tags`
+  - Resolved bug involving multiple user-defined tag structures.
+
+#### Changed attributes
+
+- `Column Naming and Ordering`
+  - Column renamed from `Column Handling`.
+- `Currency Format`
+  - Attribute renamed from `Currency Code Format`.
```

Comment:
I think we use the old name as the main bullet since that's what was changed from the previous release. I'm find either way. Just suggesting this for consistency.
```suggestion
- `Currency Code Format`
  - Attribute renamed to `Currency Format`.
```